modules/show.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable import/no-cycle */
+import removeBook from './removeBook.js';
+
+export default function Show() {
```

Comment:
- Please use ES6 arrow syntax instead of the function keyword to define functions.

Reply:
@afanimartin thank you for your review but the current time works perfectly you don't see it maybe because you didn't run [npm install]() to install the required module that is Luxon..